scripts/build_docker_image

```diff
@@ -1,19 +1,20 @@
 #!/usr/bin/env bash
 #
-# build `easi` in docker and release to ECS
+# build `easi` in docker and push to ECR
 #
 
 # log in to ECR
 if bash +x -o nounset -c "$(aws ecr get-login --no-include-email --region "us-west-2" --registry-ids "${AWS_ACCOUNT_ID}")" ; then
   # build & tag the docker image, then push to ECR
-  sha="${CIRCLE_SHA1:-"$(git rev-parse HEAD)"}"
+  APPLICATION_VERSION="${CIRCLE_SHA1:-"$(git rev-parse HEAD)"}"
+  APPLICATION_DATETIME="$(date --rfc-3339='seconds' --utc)"
+  APPLICATION_TS="$(date '+%s')"
```

Comment:
Minor point: I think you can do `date -d $APPLICATION_DATETIME '+%s'`, which will guarantee the times are the same